src/main/java/io/prestosql/plugin/tiledb/TileDBClient.java

```diff
@@ -11,16 +11,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.prestosql.plugin.tiledb;
+package io.trino.plugin.tiledb;
```

Comment:
Package names need to be named as well.

Reply:
Renamed directories, which is what I took this comment to request. Let me know if I missed something!